go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/refraction-networking/utls
+
+go 1.16
+
+require (
+	github.com/andybalholm/brotli v1.0.4
+	github.com/klauspost/compress v1.13.6
```

Comment:
These two dependencies are necessary to support Brotli and Zstandard decompression.  I added a go.mod file to allow users of this package to more readily import.